FLIR/conservator/cli/managers.py

```diff
@@ -98,6 +99,38 @@ def count(search_text):
 
     if issubclass(type_manager, CollectionManager):
 
+        @group.command(help="Create a new collection with the given path.")
+        @click.argument("collection_path")
+        @click.option(
+            "-r", "--recursive", is_flag=True, help="Create all missing path components"
+        )
+        def create_path(collection_path, recursive):
+            cpath = collection_path
+            if not cpath.startswith("/"):
+                cpath = "/" + collection_path
+            split_path = cpath.split("/")[1:]
+            manager = get_instance()
+            if len(split_path) > 1:
+                # Path is not a project.
+                base_path = "/" + "/".join(split_path[:-1])
+                # Unless recursive is set, a missing base path is an error.
+                try:
+                    collection = manager.from_string(base_path)
+                except InvalidRemotePathException:
+                    if not recursive:
+                        click.echo(f"Base path {base_path} doesn't exist.")
+                        click.echo("Specify '-r' to create all path components.")
+                        return False
+            try:
+                collection = manager.from_string(cpath)
```

Comment:
`from_remote_path` probably better here, cause it's definitely a path

Reply:
> `from_remote_path` probably better here, cause it's definitely a path

Changed.